src/lib/components/market/response_render_results_status.ts

```diff
@@ -0,0 +1,155 @@
+function overrideOnResponseRenderResults(proto: any) {
+    const originalMethod = proto.OnResponseRenderResults;
+
+    proto.OnResponseRenderResults = function(responseClass: any) {
+        const success = (responseClass && responseClass.responseJSON && responseClass.responseJSON.success) || false;
+        if (!success) {
```

Comment:
nit: Anti-pattern to do:

```
if (!var) {
   // this
} else {
   // that
}
```

instead of 

```
if (var) {
   // that
} else {
   // this
}
```